packages/xo-web/src/xo-app/vm/tab-backups.js

```diff
@@ -0,0 +1,67 @@
+import _ from 'intl'
+import addSubscriptions from 'add-subscriptions'
+import decorate from 'apply-decorators'
+import React from 'react'
+import { adminOnly } from 'utils'
+import { createPredicate } from 'value-matcher'
+import { injectState, provideState } from 'reaclette'
+import { filter, omit, some } from 'lodash'
+import { subscribeBackupNgJobs, subscribeJobs } from 'xo'
+
+import JobsTable from '../backup/overview/tab-jobs'
+
+const legacyJobKey = [
+  'continuousReplication',
+  'deltaBackup',
+  'disasterRecovery',
+  'backup',
+  'rollingSnapshot',
+]
+
+const BackupTab = decorate([
+  adminOnly,
+  addSubscriptions({
+    legacyJobs: subscribeJobs,
+    jobs: subscribeBackupNgJobs,
+  }),
+  provideState({
+    computed: {
+      haveLegacyBackups: (_, { legacyJobs }) =>
+        some(legacyJobs, job => legacyJobKey.includes(job.key)),
+      jobIds: ({ predicate }, { jobs }) =>
+        filter(jobs, predicate).map(_ => _.id),
+      predicate: (_, { vm }) => ({ vms }) =>
+        vms === undefined
```

Comment:
` vms && createPredicate(omit(vms, 'power_state'))`

Reply:
predicate is a function.

Reply:
`vms && createPredicate(omit(vms, 'power_state'))` replaces`vms === undefined ? undefined :  createPredicate(omit(vms, 'power_state'))`